server/startup/migrations/xrun.js

```diff
@@ -1,5 +1,5 @@
 if (RocketChat.Migrations.getVersion() !== 0) {
-	RocketChat.Migrations.migrateTo('latest');
+	RocketChat.Migrations.migrateTo(process.env.MIGRATION_VERSION || 'latest');
```

Comment:
yes, but it didn't work as I expected.. it needs more work before we can spread the word. :grimacing: 

for example, to run migration `97` again (avatars migration) I had to set `MIGRATION_VERSION=98,rerun` which does not make much sense :man_shrugging: